backend/tests/usersControllerTest.js

```diff
@@ -65,17 +64,32 @@ describe('Testing User Controller Class', () => {
         .send({
           "first_name": "test_staff",
           "last_name": "test_staff",
-          "email": "test_staff@gmail.com",
+          "email": "test_staff@autotracks.com",
           "role": "Management",
+          "permissions": [
+            "Add Dealerships",
+            "View Dealerships",
+            "Edit Dealerships",
+            "Delete Dealerships",
+            "Add Staff Users",
+            "View Staff Users",
+            "Edit Staff Users",
+            "Delete Staff Users",
+            "Add Vehicles",
+            "View Vehicles",
+            "Edit Vehicle Properties",
+            "Edit Vehicle Locations",
+            "Sell Vehicles",
+            "Delete Vehicles"
+        ],
           "password": "password123",
-          "dealership": "6186d06df44c867a394801dc"
+          "dealership": "618b3bf134f07d9a91c32a1b" 
         })
+        .set('authorization', 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiI2MThhYWNmNDVjZGM3NWI4Mjg4ZWI5YjUiLCJpYXQiOjE2MzgxMzc1ODUsImV4cCI6MTY0MDcyOTU4NX0.gv4Q-vPm-MqfHVsY7BYnBThxBI3bHZtUC5JtukHp340')
```

Comment:
You can create a global variable instead of using the long token every time

Reply:
This was implemented in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/34/commits/af27600b3cb8f1c93759f252d10cddda3cc79d95)